src/Components/Header/Header.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import * as ROUTES from '../../constants/routes';
+import Image from '../../pages/Login/Image/Image';
+import SvgComponent from './SvgComponent/SvgComponent';
+import PathComponent from './PathComponent/PathComponent';
+
+const Logo = {
+  src: '/images/logo.png',
+  alt: 'Instagram',
+  imageWrapperStyle: 'mt-2 w-6/12'
+};
+
+const SvgProps = {
+  className: 'w-8 mr-6 text-black-light cursor-pointer',
+  xmlns: 'http://www.w3.org/2000/svg',
+  fill: 'none',
+  viewBox: '0 0 24 24',
+  stroke: 'currentColor'
+};
+
+const PathProps1 = {
+  strokeLinecap: 'round',
+  strokeLinejoin: 'round',
+  strokeWidth: 2,
+  d: 'M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6'
+};
+
+const PathProps2 = {
+  strokeLinecap: 'round',
+  strokeLinejoin: 'round',
+  strokeWidth: 2,
+  d: 'M17 16l4-4m0 0l-4-4m4 4H7m6 4v1a3 3 0 01-3 3H6a3 3 0 01-3-3V7a3 3 0 013-3h4a3 3 0 013 3v1'
+};
+const ProfileImageProps = {
+  src: "/images/avatars/karl.jpg",
+  alt: "img profile",
+  imageWrapperStyle: "h-8 w-8 flex ",
+  imageStyle: "rounded-full"
+};
+
+export default function Header() {
+  return (
+    <header className="h-16 bg-white border-b border-gray-primary mb-8">
+      <div className="container mx-auto max-w-screen-lg h-full">
+        <div className="flex justify-between h-full">
+          <div className="text-gray-700 text-center flex items-center align-items cursor-pointer">
+            <h1 className="flex justify-center w-full">
+              <Link to={ROUTES.DASHBOARD} aria-label="Instagram logo">
+                <Image {...Logo} />
+              </Link>
+            </h1>
+          </div>
+          <div className="text-gray-700 text-center flex items-center align-items">
+            <>
+              <Link to={ROUTES.DASHBOARD} aria-label="Dashboard">
+                <SvgComponent {...SvgProps}>
+                  <PathComponent {...PathProps1} />
+                </SvgComponent>
+              </Link>
+
+              <button
+                type="button"
+                title="Sign Out"
+                onClick={() => {
+                  // Handle sign out logic here
+                }}
+                onKeyDown={(event) => {
+                  if (event.key === 'Enter') {
+                    // Handle sign out logic here
+                  }
+                }}
+              >
+                <SvgComponent {...SvgProps}>
+                  <PathComponent {...PathProps2} />
+                </SvgComponent>
```

Comment:
I prefer you create a component called SvgPathComponent, it will return:
`<SvgComponent>`
`   <PathComponent />`
`</SvgComponent>`

and it you will pass SvgProps and PathProps2 to it. 

It will be better to not duplicate this code in lines 57-59, and maybe in other places.